README.md

```diff
@@ -44,14 +44,8 @@ file as suggested [here][auto].
 ###### Windows (PowerShell)
 
 ```powershell
-md ~\vimfiles\autoload
-$uri = 'https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
-(New-Object Net.WebClient).DownloadFile(
-  $uri,
-  $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath(
-    "~\vimfiles\autoload\plug.vim"
-  )
-)
+iwr -useb https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim |`
```

Comment:
The lines ends with a pipe, `|`. Does the backtick make a difference on old powershell versions?

Reply:
The backtick is the standard escape character for every version of powershell i've used v2.0/v5.0/v7.0

This was just to make it visually look like the other commands for unix. It could also just be a single one-liner with no break.

Reply:
That's not what I meant. If a command ends with a pipe, then the shell won't execute the command immediately.

```powershell
> echo a |
>> cat
a
```

`cat` is given on the nested prompt at the next line. `echo a |` is incomplete so `echo a` does not run.

Good to know that backtick works in Powershell v2.

Reply:
Ah ok, I misunderstood, my bad.

Yes the backtick does escape the command.

It should be good either way. Especially for any version of PowerShell running by default on windows 7/8/10.